schema/v1.1.0/schema.py

```diff
@@ -136,6 +136,7 @@ def _materialize_schema(schema: SchemaView, common_schema: SchemaView) -> Schema
                             slot["description"] = common_slot["description"]
                             slot["unit"] = common_slot["unit"]
                             a["range"] = common_slot["range"]
+                            range_type = a["range"]
```

Comment:
What's this for?

Reply:
In the case that a common_slot has a range_type that is an enum, we want to update the range_type variable so that the following if statement below gets hit and the slot's pattern gets updated with enum values. I can improve the clarity of this code, if you'd like, to something like:

```python
range_type = common_slot["range"]  # update range type to ensure enums are added in the next code block
```

or

```python
if a["range"] in schema_enums:
    for _, e in enumerate(schema.get_enum(a["range"]).permissible_values):
        slot["pattern"] = _add_to_slot_pattern(slot, f"^{e}$")
```

instead of `range_type = a["range"]`.

Thank you!